internal/operator/client.go

```diff
@@ -79,29 +76,54 @@ func NewOpCapClient() (Client, error) {
 	return operatorClient, nil
 }
 
-func NewOlmClientset() (*olmclient.Clientset, error) {
-	cfg, err := clientcmd.BuildConfigFromFlags("", os.Getenv("KUBECONFIG"))
+// kubeConfig return kubernetes cluster config
+func kubeConfig() (*rest.Config, error) {
+	return ctrl.GetConfig()
+}
+
+// mustGetConfig returns only the kubernetes cluster config.
+// nil is returned when an error is encountered
+func mustGetConfig() *rest.Config {
```

Comment:
anything calling mustGet ?

Reply:
Nothing at the moment but it is there is someone does not think they need to handle the error when getting Kubernetes config. You can have it or remove it, it is there just in the event it is needed